.github/workflows/ci.yml

```diff
@@ -1176,10 +1176,6 @@ jobs:
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
-      - uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
-        with:
-          python-version: "graalpy24.1"
-
```

Comment:
do we want to *stop* testing not-uv-managed graalpy's? (genuine question)

Reply:
Definitely not.

Reply:
ah, ok, I looked at how it's done for PyPy and adapted this test to that one. But I now see there's a `system-test-pypy`, I copied that one to `system-test-graalpy` to check that.